appveyor.yml

```diff
@@ -1,9 +1,9 @@
 environment:
-  MVN_VERSION: 3.3.3
+  MVN_VERSION: 3.2.5
 
 install:
   - cinst maven --version=%MVN_VERSION% -y
-  - set PATH=C:\tools\apache-maven-%MVN_VERSION%\bin;%PATH%
+  - set PATH=C:\bin\apache-maven-%MVN_VERSION%\bin;%PATH%
```

Comment:
In the Chocolatey, maven version before 3.3.3, would be installed in _tools_ folder.